scripts/sync-version.cjs

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+const packageJsonPath = path.join( __dirname, '..', 'package.json' );
+const serverJsonPath = path.join( __dirname, '..', 'server.json' );
+
+const packageJson = JSON.parse( fs.readFileSync( packageJsonPath, 'utf8' ) );
+const serverJson = JSON.parse( fs.readFileSync( serverJsonPath, 'utf8' ) );
+
+const version = packageJson.version;
+
+serverJson.version = version;
+if ( serverJson.packages && serverJson.packages[ 0 ] ) {
+	serverJson.packages[ 0 ].version = version;
+}
+
+fs.writeFileSync( serverJsonPath, JSON.stringify( serverJson, null, 2 ) + '\n' );
+
+console.log( `✓ Updated server.json to version ${ version }` );
```

Comment:
Is this script getting called directly when running `npm version`? I'm wondering what is handling the `patch/minor/major`  values

Reply:
This script is only responsible for syncing the version from `package.json` to `server.json`. The `npm version` command is a built-in command in NPM that handles the version bump.